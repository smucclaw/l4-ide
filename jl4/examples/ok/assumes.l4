ASSUME foo IS A FUNCTION FROM NUMBER TO BOOLEAN
#EVAL foo 2

ASSUME bar IS A  BOOLEAN
baz b MEANS IF b THEN 2 ELSE 3
#EVAL baz bar
#EVAL IF bar THEN 2 ELSE 3
