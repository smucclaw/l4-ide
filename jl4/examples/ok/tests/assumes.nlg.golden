`foo` with 2
`baz` with `bar`
if `bar` then 2 else 3
